model-server-test/docker/initdb.sql

```diff
@@ -0,0 +1,251 @@
+    ('v76aw*jPYEeCgxF_vQktDoPhQoU_Vx1LnLGZAOpRqu5Q', '40000000d/mps%3A96533389-8d4c-46f2-b150-8d89155f7fca%2F2756110869689139961/400000006/requiredEquipment//%23mpsNodeId%23=5042850610501797432,equipment=2oZF3J5qvtv%2FMicrophone/'),
+    ('vdAiS*e7mNESP5krIMFnvRi4grViXUjdy6YMC5nWZoDk', 'S/6/0/6IfIy*5hy9fpez6JPK4M1_3xMaEWzVMw8OK_GLXiiJSg'),
+    ('veKbE*DGOTdYWSyisHaHMBmoLQeFhZ2FJi9k7RGXAs0c', 'L/400000003/7wX3M*XCRxTG4ZudkBCmN2gawb3-FhWIBtyvaPPmPeBY'),
+    ('w-mOf*4_tZ9EOImT57FDkOUeBELOfS4ITxvH8VDHEjHc', 'info/2/V_4I6*Hbk19Lb1sBNIU1k7aA9kGXTI3zBQqYZ9CZje74'),
+    ('wFGiQ*yr2wzRexCzit5J9QwXlAtQnaQSpiUlnTI5MDi0', 'L/100000021/dXI-I*W3Y8I3eE-HVKVf062RypZXCcuABvUJloxZl-3I'),
+    ('wKXfe*1ZXS95HuEFmhnl9XRJT-8W7Eel5id9P2eEoero', '100000003/mps%3Ab6980ebd-f01d-459d-a952-38740f6313b4%2F7113393488488348863/1/info/100000004//'),
+    ('wViE7*Dr2QLdMzP6YhAFA-8aKgx3USS1butvAODA22y0', '100000016/mps%3A96533389-8d4c-46f2-b150-8d89155f7fca%2F4128798754188010580/100000015/rooms/100000017,100000018,100000019/%23mpsNodeId%23=5042850610501964316,maximumCapacity=30,name=Marie,number=1.311/'),
+    ('ww-42*pGFDw0cKfdl6YMOb6mYH6hFQUgxo4NU4EY4ReM', '10000000e/mps%3A0a7577d1-d4e5-431d-98b1-fae38f9aee80%2F2206727074858242406/10000000d/facets//generated=true,path=%24%7Bmodule%7D%2Fclasses_gen/'),
+    ('xL0y9*PN1LAQkRi_U7ueS4ZxCT-pt1Wh4xhXW15JHE_g', '40000000a/mps%3Acafa79c7-9e66-4df7-8f06-84e4e6a7c5e6%2F5042850610501834239/400000008/endDate//%23mpsNodeId%23=5042850610501887696,date=11.03.2024/'),
+    ('xMbd7*QFsrtezEceG5xsxhhchqi9xTinTtRaYcWt1Vgs', '100000013/mps%3A0a7577d1-d4e5-431d-98b1-fae38f9aee80%2F474657388638618892/10000000d/models/100000014,100000015,10000001e,10000002c/id=r%3Ace161c54-ea76-40a6-a31d-9d7cd01febe1,name=University.Schedule.modelserver.backend.sandbox/'),
+    ('xQ_tk*pohMZohAA8ChoDUZotsSat0R5k5hSZPvwlfock', 'L/100000020/t2K0Z*2fRWwaHgE5ZJOGpbZZONBEmBpkU9yfROLSZD4M'),
+    ('xbcAG*o9JthBYkCcGFdSTvy6ygQXDTCsM4SEIHx0x09w', 'info/2/CIiES*UIqiJi7WiJCasu9yMw1nO3XzRxhvX9tWbLp01A'),
+    ('xioDt*mnraICBf48DpWkvvtl2KuPixWn1p7yteYQ3XSg', '100000017/mps%3A96533389-8d4c-46f2-b150-8d89155f7fca%2F2756110869689139961/100000016/equipment//%23mpsNodeId%23=8493110207576779271,equipment=2oZF3J5qvts%2FProjector/'),
+    ('xo8rS*7XCggPvvFZG2BhzrA3NJ-YtNdy3QucgQdNyjek', 'L/40000001c/MVmqr*dBb1Y6o0JLTKGrVOVdv6qKo82NtRi9qqEttmDs'),
+    ('yV6G2*hwGekzBG30vT5Cy43XRweqkgRM-uOCM_iKZZTI', '10000000d/mps%3A0a7577d1-d4e5-431d-98b1-fae38f9aee80%2F474657388638618895/1/modules/10000000e,10000000f,100000010,100000011,100000012,100000013/compileInMPS=true,id=a04444a1-c8a4-48e8-a940-32a70d0e8bfc,moduleVersion=0,name=University.Schedule.modelserver.backend.sandbox/'),
+    ('zK4Y2*xIEWlYlQGJL2Va4Z0ESgpWgnSQcOmnPeqt34PA', 'S/5/0/g4pFH*AQymlg23Mueo-AyIJU6n3fzGCwhSwAyOIncuFw'),
+    ('zxgZN*oLuudxsu42ppSEGnCib8LkrSvauQk2B6T7AW6o', 'courses/2/qJszC*kqdafBvlOAUkHASuNGyKiYijrmdPmA752gdjcY');
```

Comment:
Why is this included here?

Reply:
What about putting the insertion of test data into a separate file and reuse the "official" initidb.sql file from production code for setting up the database in the test. That way, we also verify that the official statements work as expected.

Reply:
This file is used for backwards compatibility tests and should put the database into an "old" state. It's not supposed to be in sync with the latest version of the "official" database.

Reply:
Then let's clarify this in the filename.

Reply:
Renamed it to `legacy-database.sql` and added a comment.